src/impersonation.tsx

```diff
@@ -0,0 +1,157 @@
+import * as React from 'react';
+import { getUser } from './session.js';
+import { signOut } from './auth.js';
+import { workos } from './workos.js';
+import { Button } from './button.js';
+import { MinMaxButton } from './min-max-button.js';
+
+interface ImpersonationProps extends React.ComponentPropsWithoutRef<'div'> {
+  side?: 'top' | 'bottom';
+}
+
+export async function Impersonation({ side = 'bottom', ...props }: ImpersonationProps) {
```

Comment:
What happens when imported in a non app router scenario?

Reply:
I haven't had the time to explore this as this was all very last minute, hence why we are in unstable `0.x.x` land for now. We'll have a ton more stuff to explore potentially.